administrator/language/de-DE/mod_latest.ini

```
@@ -15,6 +15,8 @@ MOD_LATEST_FIELD_VALUE_AUTHORS_BY_ME="Von mir erstellt oder bearbeitet"
 MOD_LATEST_FIELD_VALUE_AUTHORS_NOT_BY_ME="Nicht von mir erstellt oder bearbeitet"
 MOD_LATEST_FIELD_VALUE_ORDERING_ADDED="Zuletzt erstellte zuerst"
 MOD_LATEST_FIELD_VALUE_ORDERING_MODIFIED="Zuletzt bearbeitete zuerst"
+MOD_LATEST_HEADING_DATE_CREATED="Erstellungsdatum"
+MOD_LATEST_HEADING_DATE_MODIFIED="Bearbeitungsdatum"
```

```suggestion
MOD_LATEST_HEADING_DATE_MODIFIED="Änderungsdatum"
```

Bin mir nicht sicher ob der besser ist. In der GUI sagen wir ja auch "bearbeiten" oder "wurde bearbeitet"

---

Hatte das so aus anderen Übersetzungen kopiert.

---

Wir haben wohl hier beide Varianten im Einsatz.

---

Siehe https://github.com/joomlagerman/joomla/blob/eb755eb6b242e67acfec164fd01509b5dc4a8d35/administrator/language/de-DE/com_content.ini#L116

---

Ok dann bleibt es so :)